base_layer/core/tests/chain/chain.json

```diff
@@ -0,0 +1 @@
153bd6037f4b28ecf5bc31fd5ae6cad4b8bb739e74654c546620016dda4d08834910b2ce7b0e76bd257419eb8c015230b7120e4722fc7dd34013bfd69a94bb914731246fcb1adebaa17574dd61261b2c8c018e33dd865b3cdf8738e6fec0f85ee835cbc8ab06cd1a1ae1655fcea2455e23e037eecb47633f97faf7874bb1754a544404910004"},{"features":{"bits":0},"commitment":"3248ff7f5e2939de764102cfc9b9ad4ea659741af16501adb74506f5adc7d805","proof":"0ec92d5cab35ae2f73bafb1339b705f881eba747198f3d64967dd98f4735d5786efd9505c6bdc854b92f525ebfba149ef18a2b28b296db7123b1e353990a6b653c357160a02b811def8adc482252cc71f00bd85b97c3aa370916a33069217b1c2ce4ba52eb18caccb8263c6269899ed0d133a7cb15b930dc71efe9ead0466348187b3a1e314e161dea412664c81a3c7184a3bab224f8bf69bdbc9e31cd536c04931eb2fa7062fb24e779476b2ab8b4ce6face010d511b4dd949cc420c3efbf08261b31a4a507b3b8149d1dfb2e505355ea0b033ee4ac68c06d53f069c006ff09f88160bd62801fb1c1ced737456e7cfc0b226eb1bb26a6ef42ce91ac2630e163ca7bb35fa2c4b7e4b5287ff84b7af4ae49d6894a5b8e43a3b9759cfb17aadd7c0c8d11be7636c6fb60fdf805e0f8a8327e11a565bb7c1a74a14461eeda84222ce48cf53e42a03f52501480239b61aa5e0c573d4843d947540fad0641daa1de54a8713333a572cb30abe35d7e4e77db7e1ff03f4b582932383ee252e73f1436419ebeb5d3f1b586eecb3fe1ced1a5bd141f7bd22ca022dfbadaa3a8706c092c7f1267543fff5e59a04f4cb3426ee436ce09131323f208b5f6abf2cd7b07602607a6a3cf1af65e4223af7b3d310a74a4f231033d0270fc01bd8b4527bff0e2387b4863b83d08c2557b2f986080a6f5e7039146de487c87ac14c29eae6206e33e36e41f2baccc56ae41739c163642cca5e8bb81475a42cbc39e8fe522a9a7036570fae31b4d60aa619035d12fcbc4dc53263dfb529c122bccf3996fd7786a01903d0c47142aac5fd943a8394c99d1dcf12d4cad0d8dd935d6026f5b42a8a1325c04a9f22aa87ddfb948ad59c1dfb1124af3c8d48892c0da6a79c12126cab6c4930b99e4f4e3dab0aca80dd095358d13159a01d8fb7c453495a0ea7d932c1df72104"},{"features":{"bits":0},"commitment":"504b3ddb80a9c999525e1462a0397b9ea7e6225fbcfd25a354194af51c428c62","proof":"b88cdeaebe1f1b1781b938298bce7a829e8ba061d0d7b3129859b1798fd1be0d2e67d91c326544c43c7efee8ca664d5465b7bcc0f6ca6aacfc0a68540977740c9a82a3a4f9e68ea99161f356f0dc225d71ca0ded833924163687ea82fe8a1e7c76428268a5f172ef5628f128f1424d1cb2019fb7f2fea876db3e206b69a8c5020d7166716b0f7cdbde732d1a975e0996a6a48bad4479cb633e4b56793074840a7fb2b6d24c02dfbfa8b4a00e2848cce693455ac9a0489677919092f98f086f0837c24f28d89c9369d7ccb1673b32bd0ee51b490154e9cbcd7dde6e6abd1806020473b1e74d3c835eb31c41e1a3a82116eddac471b3f3cbb24ce119152a89824f8e3f3902b2ec0818abbf62b3e9d85137a05cd3eeb448b8514335b3c90132ee2bce4a55f372609b568ac50ef6dd5aff7ea26949cdbe84b244c962d96b23d2736b60a2d95aaf6e1af42c1d4206743c728cd3296ab7caa13c8cf643666ca88d680dd6b684725e2e8ee4faf7166d797e5d04266089e6d1468cd9eeeb667166fb8549a0dc4433b467e5f808f8e3d0a0e7663d8d2a52a41d2a121ce853dfdf0e84cb125e5ac50c30ea272e9af515c25384c26addfd879d58178df6bbf3876588c1235baaeb748b58a4a47722fc0105b7810949796ff755edb782800669cbc92a2c2745a03d0d98600ee9e12054793aacf11f17931e70664b6b6fb77420f091650c6e5a7e72b9660f688c7f92d0c7fae02d9f521c3ee1172b212fe397087ffe04aec019ac8b710c568a7660339d222e870df430ce7fe26327eb95aa5d081eba0074e11012dd5b
```

Comment:
This is one line. Can you pretty-print this?

Reply:
If we have to pretty print it, it will make deserializing difficult. Currently its just serialize -> deserialize. 
Many viewers will pretty it when viewing. Thats just the raw file.